glossary.html

```diff
@@ -4,30 +4,38 @@
     <title>sanjay’s CSD120</title>
 </head>
 <body>
+         
+    <header>
         <nav>
-        <ul>
-        <li><a href="index.html">home</a> </li>
-        <li><a href="links.html">Links</a> </li>
-        <li><a href="glossary.html">Grossary</a></li>
-        </ul>
+            <ul>
+                <li><a href="index.html">Home</a> </li>
+                <li><a href="links.html">Links</a> </li>
+                <li><a href="glossary.html">Glossary</a></li>
+            </ul>
         </nav>
+    </header>
+
+    <main>
         <h1>Acronym Glossary</h1>
-        <dt>
-        <dt>HTML</dt>
-        <dd>Hyper Text Markup Language</dd>
-        <dt>HTTP</dt>
-        <dd>Hypertext Transfer Protocol</dd>
-        <dt>IP</dt>
-        <dd>Internet Protocol</dd>
-        <dt>TCP</dt>
-        <dd>Transmission Control Protocol</dd>
-        <dt>URL</dt>
-        <dd>Uniform Resource Locator</dd>
-        </dt>
-        </main>
+        <dl>
+            <dt>HTML</dt>
+                <dd>Hyper Text Markup Language</dd>
+            <dt>HTTP</dt>
+                <dd>Hypertext Transfer Protocol</dd>
+            <dt>IP</dt>
+                <dd>Internet Protocol</dd>
+            <dt>TCP</dt>
+                <dd>Transmission Control Protocol</dd>
+            <dt>URL</dt>
+                <dd>Uniform Resource Locator</dd>
+        </dl>
+    </main>
+    
+    <footer>
         <hr>
         <blockquote>"Just because something doesn&#39;t do what you planned it to do doesn&#39;t mean it&#39;s useless".<br>
-        <em>&#9472;Thomas Edison</em></blockquote>
-        <p>&copy; 2019 <a href="mailto:19046617@saultcollege.ca?subject=Message from CSD120 Website">sanjay</p>
+        <em>&mdash;Thomas Edison</em></blockquote>
```

Comment:
The dash you have here isn't really the em dash, but I won't remove marks for this